src/engine/server.h

```diff
@@ -251,10 +251,11 @@ class IServer : public IInterface
 		RCON_CID_SERV = -1,
 		RCON_CID_VOTE = -2,
 	};
+	virtual class CRconRole *RoleOrNullptr(int ClientId) = 0;
```

Comment:
```suggestion
	virtual CRconRole *RoleOrNullptr(int ClientId) = 0;
```
If necessary, forward declare class above `IServer`.